src/components/Search/Search.js

```diff
@@ -16,8 +16,8 @@ const Search = ({ determineSearchResults }) => {
   }
 
   const filterPosts = () => {
-    const postsToDisplay = allPosts.filter(post => {
-      return post.title.includes(searchInput) || post.description.includes(searchInput)
+    const postsToDisplay = allData[0].getAllPosts.filter(post => {
+      return post.title.toLowerCase().includes(searchInput) || post.description.toLowerCase().includes(searchInput)
```

Comment:
Really good addition!

Reply:
yes, definitely a helpful update for our search!